docs/api-docs/management-apis/email-templates/email-template-examples.md

```diff
@@ -0,0 +1,116 @@
+# Email Templates Code Examples
+
+<div class="otp" id="no-index">
+
+### On this page
+
+- [Updating text](#updating-text)
+- [Adding a button](#adding-a-button)
+- [Changing logo size](#changing-logo-size)
+- [Changing font size](#changing-font-size)
+
+</div>
+
+This article provides examples of how to customize email templates. The examples are for use in the recently updated templates. You can enable the new version of transactional emails by going to **Marketing> Transactional Emails** and clicking **Try New Experience**. When you are ready to use a template, make sure you **Enable** the template from the respective dropdown. 
+
+Code examples below will demonstrate how to update text, add a button, change logo size, and change the font size.
+
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+  
+  ### Note 
+  It is important to preview all changes and style your emails for accessibility.  
```

Comment:
I feel like this note doesn't provide me with sufficient information to take action. Perhaps you can link to the [Developing Accessible Themes](https://developer.bigcommerce.com/stencil-docs/accessibility/theme-accessibility) article to give readers more context.